locust/main.py

```diff
@@ -23,6 +23,7 @@
 
 _internals = [Locust, HttpLocust]
 version = locust.__version__
+global_repetition = 1
```

Comment:
need global variable of is_integration (boolean) as well

Reply:
the global_repetition value must be overridden by -t value
and the is_integration value must be overridden  by --integration